src/appengine/handlers/cron/backup.py

```diff
@@ -80,6 +81,4 @@ def get(self):
       status_code = e.resp.status
       logs.log_error(message, error=str(e))
 
-    self.response.headers['Content-Type'] = 'text/plain'
-    self.response.out.write(message)
-    self.response.set_status(status_code)
+    return make_response(message, status_code, {'Content-Type': 'text/plain'})
```

Comment:
nit: turns out the make_response call isn't actually necessary. You can just return the tuple.